kytos/core/kytosd.py

```diff
@@ -84,6 +98,12 @@ def start_shell(controller=None):
 
 def main():
     """Read config and start Kytos in foreground or daemon mode."""
+    # data_files is not enough when installing from PyPI
+    for file in ETC_FILES:
+        shutil.copy2(file, Path(BASE_ENV) / file)
```

Comment:
What's the purpose of this when ETC_FILES is just an empty list?

Reply:
I'm not sure, but It's something that already were in the project, I'll investigate it to understand. For now, I don't see any reason to keep that.

Reply:
It's possible to check that the ETC_FILES got an empty list in commit c4b1a1a9, and stayed empty since then. I just removed this code and tested the installation. Everything seems to work fine.